recipes/continuable/all/conanfile.py

```diff
@@ -1,80 +1,118 @@
 import os
 
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
 
-
-required_conan_version = ">=1.28.0"
+required_conan_version = ">=1.52.0"
 
 
 class ContinuableConan(ConanFile):
     name = "continuable"
-    description = "C++14 asynchronous allocation aware futures (supporting then, exception handling, coroutines and connections)"
-    topics = "asynchronous", "future", "coroutines", "header-only"
+    description = (
+        "C++14 asynchronous allocation aware futures "
+        "(supporting then, exception handling, coroutines and connections)"
+    )
+    license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/Naios/continuable"
-    license = "MIT"
-    settings = "os", "compiler"
-    no_copy_source = True
-    requires = (
-        "function2/4.1.0",
-    )
+    topics = ("asynchronous", "future", "coroutines", "header-only")
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
     options = {
-        # Exceptions are disabled and `std::error_condition` is used as error_type. See tutorial-chaining-continuables-fail for details.
         "no_exceptions": [True, False],
-        # Exceptions are disabled and the type defined by `CONTINUABLE_WITH_CUSTOM_ERROR_TYPE` is used as error_type.
-        # See tutorial-chaining-continuables-fail for details.
         "custom_error_type": [True, False],
-        # Allows unhandled exceptions in asynchronous call hierarchies. See tutorial-chaining-continuables-fail for details.
         "unhandled_exceptions": [True, False],
-        # Allows to customize the final callback which can be used to implement custom unhandled asynchronous exception handlers.
         "custom_final_callback": [True, False],
-        # Don"t decorate the used type erasure, which is done to keep type names minimal for better error messages in debug builds.
         "immediate_types": [True, False],
     }
     default_options = {
         "no_exceptions": False,
         "custom_error_type": False,
         "unhandled_exceptions": False,
         "custom_final_callback": False,
-        "immediate_types": False
+        "immediate_types": False,
+    }
+    options_description = {
```

Comment:
I _love_ this! thanks for moving the comments to a place where now the user can see it easily :)